src/containers/DataTable/index.js

```diff
@@ -70,14 +71,32 @@ export function DataTable ({
 							<i className={title.icon}></i> : title.icon
 						}
 						{title.text}
+						{button && 
+							<div className="float-right ml-3 data-table-create-button">
+								<Button
+									tag="span"
+									size="sm"
+									{...button?.props}
+								>
+									{button.icon && 
+										<span className="pr-1">
+											{typeof button.icon === 'string' ? 
+												<i className={button.icon}></i> : button.icon
+											}
+										</span>
+									}
+									{button?.text}
+								</Button>
+							</div>
+						}
 						{createButton &&
 							<div className="float-right ml-3 data-table-create-button">
 								<Link to={{ pathname: createButton.pathname }}>
 									<Button tag="span" size="sm">
 										{createButton.icon && 
-										<span className="pr-1">
-											{typeof createButton.icon === 'string' ? <i className={createButton.icon}></i> : createButton.icon}
-										</span>
+											<span className="pr-1">
+												{typeof createButton.icon === 'string' ? <i className={createButton.icon}></i> : createButton.icon}
+											</span>
```

Comment:
I think indents are not correct here, because GitHub makes them green

Reply:
They were not correct because they were aligned with parent curly brackets, I changed them on purpose.